tests/DancePage.py

```diff
@@ -11,11 +12,12 @@
 
 # Wrapper around an appium interface to a Dance Party page for readable tests
 class DancePage:
-    def __init__(self, origin='https://test-studio.code.org/'):
```

Comment:
Is going from `studio.code.org` to `test-studio.code.org` necessary for this change? If not, are we comfortable about testing on prod rather than test?

Reply:
Sorry, I accidentally included this change, its not necessary. What do you think is the best default? I can change it back to test if we prefer that.

Reply:
Thought about this a bit more and IMO prod is a better default since it's less likely to be disrupted by a deploy, and also since these stress tests are run out-of-band from the deployment process, so they're not necessarily testing the current new DTT. It's easy to change back to staging/test if needed for a feature that hasn't been DTP'd yet and if we find ourselves doing this often, its not hard to add a CLI argument for this either.

Reply:
Sounds like a good plan to me. :+1:

Reply:
A downside of pointing these stress tests at production is that it's got some (small but nonzero) impact on our metrics.  I don't think we've currently got any way to filter out activity generated by this automated tool when we measure classroom activity, like total page loads for puzzle 10 or average time engaged with free-play mode.

In practice I'm not sure we run these often enough to make a dent, but I'd feel a little better pointing it at production if we were using an allthethings level instead of one in the normal script progression.